pkg/components/util/install.go

```diff
@@ -59,6 +61,46 @@ func ensureNamespaceExists(name string, kubeconfigPath string) error {
 	return nil
 }
 
+// DisableAutomountServiceAccountToken updates default Service Account to not mount it in pods by default.
+func DisableAutomountServiceAccountToken(ns, kubeconfigPath string) error {
+	if ns == "" {
+		return fmt.Errorf("namespace name can't be empty")
+	}
+
+	kubeconfig, err := ioutil.ReadFile(kubeconfigPath) // #nosec G304
+	if err != nil {
+		return fmt.Errorf("reading kubeconfig file: %w", err)
+	}
+
+	cs, err := k8sutil.NewClientset(kubeconfig)
+	if err != nil {
+		return fmt.Errorf("creating clientset: %w", err)
+	}
+
+	payload := []patchUInt32Value{{
+		Op:    "add",
+		Path:  "/automountServiceAccountToken",
+		Value: false,
+	}}
+
+	payloadBytes, _ := json.Marshal(payload)
+
+	ctx := context.TODO()
+	sa, _ := cs.CoreV1().ServiceAccounts(ns).Get(ctx, "default", metav1.GetOptions{})
+
+	automountServiceAccountToken := sa.AutomountServiceAccountToken
+
+	if automountServiceAccountToken == nil {
```

Comment:
IIUC with this code if `automountServiceAccountToken` is set to true, it will not touch it.

Should we always set it to false?

AFAIU using the `add` operation for `Patch` will replace the object if it exists so I think we can remove this check. We should make sure it works as expected, though.